misk/src/main/kotlin/misk/MiskTestingServiceModule.kt

```diff
@@ -18,24 +18,51 @@ import com.google.inject.spi.InstanceBinding
 import com.google.inject.spi.LinkedKeyBinding
 import com.google.inject.spi.ProviderInstanceBinding
 import com.google.inject.util.Types
+import misk.environment.FakeEnvVarModule
+import misk.environment.RealEnvVarModule
 import misk.healthchecks.HealthCheck
 import misk.inject.KAbstractModule
 import misk.metrics.MetricsModule
 import misk.moshi.MoshiModule
 import misk.prometheus.PrometheusHistogramRegistryModule
 import misk.resources.ResourceLoaderModule
+import misk.resources.TestingResourceLoaderModule
 import misk.time.ClockModule
 import misk.tokens.TokenGeneratorModule
 import javax.inject.Singleton
 
-class MiskServiceModule : KAbstractModule() {
+/**
+ * [MiskRealServiceModule] should be installed in real environments.
```

Comment:
this is the real change. the rest is just changing existing code to use either Real or Testing